python/py-bracex/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-bracex
+version             2.1.1
+revision            0
+python.versions     36 37 38 39
+
+platforms           darwin
+license             MIT
+maintainers         outlook.de:judaew openmaintainer
+
+description         Bash style brace expansion for Python.
+long_description    ${description} Brace expanding is used to generate \
+                    arbitrary strings.
+homepage            https://github.com/facelessuser/wcmatch
```

Comment:
this homepage doesn't seem correct... or?

Reply:
Thanks, yes is it.